sockjs/frame.go

```diff
@@ -1,11 +1,41 @@
 package sockjs
 
 import (
-	"encoding/json"
-	"fmt"
+	"bytes"
+	"strconv"
 )
 
 func closeFrame(status uint32, reason string) string {
-	bytes, _ := json.Marshal([]interface{}{status, reason})
-	return fmt.Sprintf("c%s", string(bytes))
+	return "c[" + strconv.FormatUint(uint64(status), 10) + "," + strconv.Quote(reason) + "]"
+}
+
+func frame(messages []string) string {
+	switch len(messages) {
+	case 0:
+		return ""
+	case 1:
+		return messageFrame(messages[0])
+	default:
+		return arrayFrame(messages)
+	}
+}
+
+func messageFrame(message string) string {
+	return "m" + strconv.Quote(message)
```

Comment:
I am not sure that `strconv.Quote` does the same as `json.Marshal` before. I experimented with 671KB json file:

```
wget "https://raw.githubusercontent.com/FZambia/easyerror/master/sample.json" -O sample.json
```

And running this code:

```go
package main

import (
	"encoding/json"
	"io/ioutil"
	"strconv"
)

func quote(in string) string {
	quoted, _ := json.Marshal(in)
	return string(quoted)
}

func main() {
	data, _ := ioutil.ReadFile("sample.json")

        // before
	d := quote(string(data))
	var m interface{}
	err := json.Unmarshal([]byte(d), &m)
	if err != nil {
		println("Error:", err.Error())
	} else {
		println("No problem with json.Marshal")
	}

        // after
	s := strconv.Quote(string(data))
	var f interface{}
	err = json.Unmarshal([]byte(s), &f)
	if err != nil {
		println("Error:", err.Error())
	} else {
		println("No problem with strconv.Quote")
	}
}
```

json.Unmarshal fails for data encoded using `strconv.Quote`